release/README.md

```diff
@@ -0,0 +1,15 @@
+## EKS Anywhere Curated Packages Release Runbook
+
+The steps to create a new release of EKS-A Package Controller
+
+1. Tag the repository for example `git tag v0.2.2`
+1. Push the tag `git push :v0.2.2`
+1. Update the build tooling:
+    * Update the GIT_TAG https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/aws/eks-anywhere-packages/GIT_TAG
+    * Run `make release` and update the CHECKSUMS file
+    * Update the release in the README
+    * Create a pull request and get it merged
+1. Update the BUNDLE_NUMBER in https://github.com/aws/eks-anywhere/tree/release-0.11/release/triggers/bundle-release
```

Comment:
```suggestion
1. Update the BUNDLE_NUMBER in https://github.com/aws/eks-anywhere/tree/release-<RELEASE-NUMBER>/release/triggers/bundle-release
```
Just to prevent a cut and paste error?

Reply:
We shouldn't be the one's upgrading this number out of band ever, it's done via the release lead during the release.

Reply:
The bundle release shouldn't be tied to the CLI release though. We should be able to create a new bundle at any time.

Reply:
That would kick off a new Prod bundle release, which would override the bundle in the s3 bucket that customer's would be using. I'd doublecheck with Jackson, but almost certain we shouldn't be doing this except on patches or releases.

Reply:
overriding the bundle sounds to me exactly what we'd want to do